README.md

```diff
@@ -159,6 +159,7 @@ If you are looking to contribute then explore this list, look at [`first-timers-
 
 - [TypeScript](https://github.com/Microsoft/TypeScript/labels/Effort%3A%20Easy) _(label: Effort: Easy)_ TypeScript is a superset of JavaScript that compiles to clean JavaScript output.
 - [Visual Studio Code](https://github.com/Microsoft/vscode/labels/good%20first%20issue) _(label: good first issue)_ a new type of tool that combines the simplicity of a code editor with what developers need for their core edit-build-debug cycle.
+- [Planet Learning](https://github.com/ole-vi/planet) _(label: Newbie Friendly)_ A distributed learning platform for community especially in place where internet is a constraint.
```

Comment:
Please update link to the labeled issues

Reply:
Done @vvscode